pages/ksphere/kommander/latest/clusters/attach-cluster/index.md

```diff
@@ -8,13 +8,17 @@ excerpt: A guide for attaching an existing Kubernetes cluster using kubeconfig
 
 ## Attach Kubernetes Cluster
 
-Using the **Add Cluster** option, you can attach an already existing cluster, or an existing Konvoy cluster, directly to Kommander. You can access the multi-cluster management and monitoring benefits Kommander provides while keeping your existing cluster on its current provider and infrastructure.
+You can attach an existing cluster directly to Kommander. There are two main ways of doing it, described below, and one requires to create a new service account. If the cluster you wish to attach has been created using Amazon EKS, Azure AKS, or Google GKE, please follow the instructions "Attaching Kubernetes Cluster Using a New Service Accoun".
 
-Selecting the **Connect Cluster** option displays the **Connection Information** dialog box. This dialog box accepts a kubeconfig file, that you can paste, or upload into the field. In the **Conext** select list, you can select the intended context or change the display name provided with the config. You can add labels to classify your cluster.
+### Attaching a Kubernetes Cluster
```

Comment:
This is basically the same heading as the `##` one. Also, this section describes the exact same steps that are necessary for every cluster, even those that I need to create an SA for beforehand.

From my perspective it would make sense to move the `Attaching Kubernetes Cluster Using a New Service Account` section before this one and mark it optional somehow.